### ISSUE TITLE
Can't change "Indoor target temp." in the Lovelace card

### ISSUE THREAD

**Issue by barvidss** [ThermIQ/thermiq_mqtt-ha#20]
## Version of the custom_component
v.1.2.3

## Configuration

```yaml

thermiq_mqtt:
  mqtt_node: ThermIQ/ThermIQ-mqtt-bb
  thermiq_dbg: False

```

## Describe the bug
When changing the value of "Indoor target temp." in the Lovalace card nothing changes on the pump and after a while the value in the card is reverted back to the original value.

## Debug log

Nothing in the log
```text



```

**Comment by ThermIQ** [comment]
Please check what messages are sent and received by using MQTT-Explorer and post these here

**Comment by ThermIQ** [comment]
Should be resolved in v1.2.4
